baseplate/integration/wrapped_context.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+
+class WrappedRequestContext(object):
+    def __init__(self, context, trace=None):
+        self.__dict__['_context'] = context
+        self.__dict__['logger'] = logging.getLogger(self.__class__.__name__)
+
+    def __getattr__(self, attr):
+        return getattr(self._context, attr)
+
+    def __setattr__(self, attr, value):
+        if attr in self._context.__dict__:
+            self.logger.debug("Assigning attr=%s of wrapped request context.")
```

Comment:
This might be a bit spammy for running apps in debug mode. Do you think it's generally useful?